sw/device/lib/dif/meson.build

```diff
@@ -0,0 +1,16 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# PLIC DIF library (dif_plic)
+dif_plic = declare_dependency(
+  link_with: static_library(
+    'dif_plic_ot',
+    sources: [
+      'dif_plic.c',
+    ],
+    dependencies: [
+      sw_lib_mmio,
+    ]
```

Comment:
Put one element lists on one line, like `sources: ['dif_plic.c']`. We're mostly consistent about this style in our Meson files.